chore-core.gemspec

```diff
@@ -41,4 +41,5 @@ Gem::Specification.new do |s|
   s.add_development_dependency(%q<rspec>, [">= 3"])
   s.add_development_dependency(%q<dalli>, [">= 2"])
   s.add_development_dependency(%q<timecop>, [">= 0"])
+  s.add_development_dependency(%q<ffi>, ["< 1.17"])
```

Comment:
Had to lock this due to issues on gem build. 
https://github.com/ffi/ffi/issues/1103#issuecomment-2186974923